src/datumaro/plugins/data_formats/coco/page_mapper.py

```diff
@@ -118,14 +131,15 @@ class COCOPageMapper:
     in stream manner after constructing the page map.
     """
 
-    _n_chars = 1024 * 1024 * 16  # 16MB
+    # _n_chars = 1024 * 1024 * 16  # 16MB
+    _n_chars = 1024 * 64  # 64KB
     cnt = 0
 
     def __init__(self, path: str):
         self.path = path
         self.section_offsets = {
             section: self._find_section_offset(section)
-            for section in [COCOSection.IMAGES, COCOSection.ANNOTATIONS]
+            for section in [COCOSection.IMAGES, COCOSection.ANNOTATIONS, COCOSection.CATEGORIES]
```

Comment:
I am sorry that I missed to find `categories` in the COCO Json format. Could you explain this?

Reply:
In the actual COCO2017 instance dataset, "categories" are far behind in the JSON file.

```
{
    "images: ...,
    "annotations: ...,
    "categories": ...
```

This makes `json-stream` parse to the end of JSON file and this makes it slow.

Reply:
I found it. thanks.